helm/values.yaml

```diff
@@ -63,9 +63,54 @@ policy:
   - pattern: "docker.io/securesystemsengineering/connaisseur:*"
     verify: false
 
+
 # in detection mode, deployment will not be denied, but only prompted
 # and logged. This allows testing the functionality without
 # interrupting operation.
 detection_mode: false
 
 # debug: true
+
+# alerting is implemented in form of simple POST requests with json payload
+# you can use and/or adapt the predefined Slack/OpsGenie/Keybase templates and the examples below
+# to channel alert notifications to Slack/OpsGenie/Keybase or create a custom template for a customized alert
+# payload to use with a simple POST request to the receiver_url to receive alerts.
+# Parameters you can use in your templates are "alert_message", "priority", "connaisseur_pod_id", "cluster",
+# "timestamp", "request_id" and "images" each one basically meaning what their names indicate
+# Below is an example config
+
+alerting:
+  config_dir_path: "/app/config"
+  cluster_name: "minikube"
+#  admit_request:
+#    message: "CONNAISSEUR admitted a request"
+#    templates:
+#      # <template> needs to be chosen such that <template>.json matches one of the file names
+#      # in the ./alert_payload_templates directory
+#      - template: opsgenie #REQUIRED!
+#        receiver_url: https://api.eu.opsgenie.com/v2/alerts #REQUIRED!
+#        priority: 4 #(defaults to 3)
+#        custom_headers: ["Authorization: GenieKey 12345678-abcd-2222-3333-1234567890ef"]
```

Comment:
hm. is it an issue that this key is "publically" available? anyone with configmap read rights could send out fake alerting messages. 

problem also would be, that we don't know which of the field contains sensitive information. in opsgenie's case it's the `custom_headers`, for slack and keybase it's part of the receiver url.

Reply:
Good point!! As you're right about that we don't really know where the secret part will be, I should probably turn the whole `alertconfig.json` into a secret, what do you think?

Reply:
yea. it's not the best solution, but i can't think of a better one right now. so let's do the whole alerting config as secret for now.

Reply:
:+1: